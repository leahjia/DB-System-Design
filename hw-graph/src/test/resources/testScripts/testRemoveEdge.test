# University Map Edge Removal testing file.
# Tests that an edge can be removed from the university map

# Remove a pair of edges
RemoveEdge e1
RemoveEdge e2
