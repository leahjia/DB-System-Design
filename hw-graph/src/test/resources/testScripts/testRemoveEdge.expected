# University Map Edge Removal testing file.
# Tests that an edge can be removed from the university map

# Remove a pair of edges
removed edge e1 from map1
removed edge e2 from map1
